src/stdlib/strings.bpf.c

```diff
@@ -218,8 +218,780 @@ m_str* __strerror(int errno, m_arg *out) {
   if (errno >= 0 && errno <= EHWPOISON) {
     result = &errors[errno];
   } else {
-    result = &unknown;
+    result = &unknown_error;
   }
   __builtin_memcpy(&out->data, result, sizeof(*out));
   return &out->data;
 }
+
+// Syscall table for Linux x86_64. Automatically generated from
```

Comment:
Automatically generated from ...? :)

I think it's fine to have this to be static, given that it is sacred and part of the stable ABI. But it would be good to include how it was originally generated, just to give some context.